app/src/main/java/rick_and_morty/ui/character_details/composables/CharacterDetailsRow.kt

```diff
@@ -0,0 +1,93 @@
+package rick_and_morty.ui.character_details.composables
+
+import android.util.Log
+import androidx.compose.animation.core.FastOutSlowInEasing
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.rememberImagePainter
+import rick_and_morty.data.model.CharacterDetails
+import rick_and_morty.data.model.CharacterResultsDto
+
+@Composable
+fun CharacterDetailsRow(characterResultsDto: CharacterResultsDto, modifier: Modifier = Modifier) {
+    val imagePainter = rememberImagePainter(data = characterResultsDto.image)
+    val imageSize by animateFloatAsState(
+        targetValue = 0.5f,
+        animationSpec = tween(
+            durationMillis = 300,
+            delayMillis = 50,
+            easing = FastOutSlowInEasing
+        )
+    )
+    val characterDetailsList = listOf(
+        CharacterDetails("Name", characterResultsDto.name),
+        CharacterDetails("Last known location", characterResultsDto.locationDto.name),
+        CharacterDetails("Species", characterResultsDto.species),
+        CharacterDetails("Created", characterResultsDto.created.substring(0, 10)),
+        CharacterDetails("Gender", characterResultsDto.gender),
+        CharacterDetails("Origin", characterResultsDto.originDto.name),
+        CharacterDetails("Status", characterResultsDto.status)
+    )
```

Comment:
this logic belongs in the VM so it can be tested. The View renders data. The VM transforms it.

Reply:
Done, please check how it looks @LordRaydenMK